tests/phpunit/elementor/includes/template-library/test-manager-general.php

```diff
@@ -172,12 +215,14 @@ public function test_should_return_wp_error_template_error_from_export_template(
 		);
 	}
 
-	public function test_get_templates() {
+	public function test_get_templates_ff() {
 		// Arrange
 		$admin = $this->act_as_admin();
 
 		$document_ids = $this->create_mock_templates( $admin, 'container_flexbox' );
 
+		$this->sourceCloudMock->method('get_items')->willReturn([]);
```

Comment:
```suggestion
		$this->sourceCloudMock->method( 'get_items' )->willReturn( [] );
```
Not sure why the linter does not catch it, but please check all the test files.